scripts/Muon/GUI/Common/results_tab_widget/results_tab_view.py

```diff
@@ -0,0 +1,175 @@
+        function_selector.blockSignals(True)
+        function_selector.clear()
+        function_selector.addItems(names)
+        if original_selection:
+            function_selector.setCurrentText(original_selection)
+        function_selector.blockSignals(False)
+
+    def selected_result_workspaces(self):
+        """
+        :return: The list of selected workspaces and their positions in the list
+        """
+        return self.fit_selector_presenter.get_selected_items_and_positions()
+
+    def fit_result_workspaces(self):
+        """
+        :return: The current state of the workspace list selector
+        """
+        return self.fit_selector_presenter.model
+
+    def set_fit_result_workspaces(self, workspace_list_state):
+        """Set the map of workspaces
+
+        :param workspace_list_state: Dictionary containing the updated
+        state for the workspace list selector
+        """
+        self.fit_selector_presenter.update_model(workspace_list_state)
+
+    def selected_fit_function(self):
+        """Return the text of the selected item in the function
+        selection box"""
+        return self.fit_function_selector.currentText()
+
+    def selected_log_values(self):
+        """
+        :return: A list of selected log values and their positions in the list
+        """
+        return self.log_selector_presenter.get_selected_items()
+
+    def log_values(self):
+        """
+        :return: The current state of the log list selector
+        """
+        return self.log_selector_presenter.model
+
+    def set_log_values(self, logs_list_state):
+        """Set the map of log values and selected status
+
+        :param logs_list_state: Dictionary containing the updated
+        state for the workspace list selector
+        """
+        self.log_selector_presenter.update_model(logs_list_state)
+
+    # Private methods
+    def _init_layout(self):
+        """Setup the layout of the view"""
+        self.setupUi(self)
+
+        self.log_selector_presenter = _create_empty_list_selector(
+            self, LOG_SELECTOR_COL0_WIDTH)
+        self.log_value_layout.addWidget(self.log_selector_presenter.view,
+                                        *FIT_SELECTOR_GRID_POS)
+        self.fit_selector_presenter = _create_empty_list_selector(
+            self, FIT_SELECTOR_COL0_WIDTH)
+        self.fit_layout.addWidget(self.fit_selector_presenter.view,
+                                  *LOG_SELECTOR_GRID_POS)
+
+    def _init_signals(self):
+        """Connect internal signals to external notifiers"""
+        self.fit_function_selector.currentIndexChanged.connect(
+            self.function_selection_changed)
+        self.results_name_editor.editingFinished.connect(
+            self.results_name_edited)
+        self.output_results_table_btn.clicked.connect(
+            self.output_results_requested)
+
+    # def update_fit_function_list(self, fit_function_list):
```

Comment:
is this commented code suppose to be here?

Reply:
Nice catch. No that was some old code I missed removing.